Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -271,82 +271,29 @@ function New-LDAPIdentitySource {
     }
 }
 
-<#
-    .Synopsis
-     Download the certificate from a domain controller
-#>
-function Get-CertificateFromDomainController {
-    param (
-        [Parameter(
-            Mandatory = $true)]
-        [ValidateNotNull()]
-        [System.Uri]
-        $ParsedUrl,
-
-        [Parameter(
-            Mandatory = $true)]
-        [ValidateNotNull()]
-        [string]
-        $computerUrl
-    )
-
-    try {
-        $Command = 'nslookup ' + $ParsedUrl.Host + ' -type=soa'
-        $SSHRes = Invoke-SSHCommand -Command $Command -SSHSession $SSH_Sessions['VC'].Value
-        if ($SSHRes.ExitStatus -ne 0) {
-            throw "The FQDN $($ParsedUrl.Host) cannot be resolved to an IP address. Make sure DNS is configured."
-        }
-
-        $Command = 'nc -vz ' + $ParsedUrl.Host + ' ' + $ParsedUrl.Port
-        $SSHRes = Invoke-SSHCommand -Command $Command -SSHSession $SSH_Sessions['VC'].Value
-        if ($SSHRes.ExitStatus -ne 0) {
-            throw "The connection cannot be established. Please check the address, routing and/or firewall and make sure port $($ParsedUrl.Port) is open."
-        }
-
-        Write-Host ("Starting to Download Cert from " + $computerUrl)
-        $Command = 'echo "1" | openssl s_client -connect ' + $ParsedUrl.Host + ':' + $ParsedUrl.Port + ' -showcerts'
-        $SSHRes = Invoke-SSHCommand -Command $Command -SSHSession $SSH_Sessions['VC'].Value
-        $SSHOutput = $SSHRes.Output | out-string
-    } catch {
-        throw "Failure to download the certificate from $computerUrl. $_"
-    }
-    return $SSHOutput
-}
-
 <#
     .Synopsis
      Recommended: Add a secure external identity source (Active Directory over LDAPS) for use with vCenter Server Single Sign-On.
-
```

Comment:
Why did those disappear, please leave the empty line.

Reply:
should be good now.